homeassistant/components/habitica/util.py

```diff
@@ -139,3 +144,18 @@ def get_recurrence_rule(recurrence: rrule) -> str:
 
     """
     return str(recurrence).split("RRULE:")[1]
+
+
+def get_config_entry(hass: HomeAssistant, entry_id: str) -> HabiticaConfigEntry:
```

Comment:
Since this is only used in services.py, and it is directly linked to service (ie. it raises ServiceValidationError), I'm not sure if makes sense to have it in a separate module.

Reply:
yep, good point, i'll move it to services.py